run.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPTPATH=`cd $(dirname "$0") && pwd`
+if [ ! -d $SCRIPT_PATH ]; then
+  echo "Could not determine absolute dir of $0"
+  echo "Maybe accessed with symlink"
+fi
+BENCHMARKS="$SCRIPTPATH"
+
+RIR_VM=$(readlink -f $1)
```

Comment:
This does not work in MacOS. Use this instead if OS != linux
https://github.com/reactorlabs/rir/blob/master/tools/downloadBenchs.sh#L53